release-team/role-handbooks/release-notes/README.md

```diff
@@ -75,9 +77,11 @@ As well as becoming a member of the kubernetes GitHub organization as discussed
 ### Machine and GitHub Setup
 
 #### Setup krel
+
 Install Go in your machine and follow the [instructions to build the release tools](https://github.com/kubernetes/release/tree/master/docs/krel#installation) in your machine. Check the system requirements in the krel documentation.   
 
 #### Get a GitHub token
+
 4. Obtain GitHub API Token with the repo scope:
 https://github.com/settings/tokens
 - [X] repo
```

Comment:
Comments: 

- @JamesLaverack showed an API token with far higher permissions than described in the doc here during their demo. Is this accurate still?
- Nit: s/API Token/Personal Access Token (follow the same wording as GitHub's UI to reduce confusion for first time readers.

Reply:
Tried `krel release-notes` with a key with these perms just now and it failed with:

```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

Which makes sense, these perms don't give read/write 🙃. `echo $GITHUB_TOKEN` works correctly though?

Reply:
I think the permissions listed work, no need to have more. I'm not 100% sure if they're the bare minimum though. I changed  API Token to Personal Access Token.